react-components/playingWithComponents/src/CustomButton.tsx

```diff
@@ -0,0 +1,7 @@
+import React from 'react';
+
+function CustomButton() {
+  return <button>Click Me</button>;
+}
+
+export default CustomButton;
```

Comment:
I would be very happy if you always used the `export function CustomButton()` syntax.